apps/api/src/modules/packs/index.ts

```diff
@@ -208,6 +210,23 @@ export async function packsRoutes(app: FastifyInstance) {
     mintPackStatus
   )
 
+  app.post(
```

Comment:
This is an admin-only endpoint and it's living alongside end-user endpoints, I don't see a check to make sure the  the caller is an admin (but I def could be overlooking it). Do we trust the web layer to do the [appropriate authorization](https://github.com/deptagency/algomart/pull/271/files#diff-fbddb1ab07ada423622f6f4c479e2ca80410238045d7d4209c2083835cde88e9R15) or does the API layer need it's own authorization?

Reply:
Currently all of the admin-specific endpoints are living among the other endpoints, and we are trusting the web layer to do the appropriate authorization. There is no concept of an admin role in the API, since Firebase Admin isn't used in the API and that's what we're using for determining permissions. It's definitely something to continue considering, if we want to add authorization to the API layer, but it didn't seem necessary at this time. Typically you would implement Firebase Admin in the API, but since the front-end has it's own API server, we can do the authorization on those endpoints using the new adminMiddleware instead. This seemed secure, as this verification needs to happen in order to call the endpoints, but ideally the API would determine this.

Reply:
@dallashuggins this one's for you.

Reply:
makes sense, thanks for explaining!